gustaf/helpers/data.py

```diff
@@ -209,10 +205,6 @@ def __setitem__(self, key, value):
         -----------
         key: str
         value: object
-
-        Returns
-        --------
-        None
```

Comment:
How else do you indicate that its a void function

Reply:
That there is no returns in the doc string. The Returns None was also much less the problem than the Attributes None